src/Datadog.Trace.Build/Datadog.Trace.Build.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.18.4-prerelease</Version>
+  </PropertyGroup>
+
+  <!-- For VS testing purposes only, copy all implementation assemblies to the
+       to test the loading of managed profiler assemblies from disk -->
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and '$(TargetFramework)' == 'netstandard2.0'">
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="14.3.0" />
```

Comment:
Not blocking, what is the earliest MSBuild version / version of the `Microsoft.Build.Framework` assembly that  we intend to support? I noticed in the GAC (maybe installed by Visual Studio) that there is a version of `Microsoft.Build.Framework` 4.0.0.0

Reply:
I did not realized that `Microsoft.Build.Framework.dll` was included as part of .NET Framework for 4.5 and later, I've just installed a vm with win8.1 with .NET Framework 4.5 and I can see the file in the GAC, so I've changed the project to use the Reference for .NET Framework and the nuget for netcore.

![image](https://user-images.githubusercontent.com/69803/89191374-1eb84600-d5a3-11ea-96b8-2ca1968cf871.png)

Reply:
as a reference: https://apisof.net/catalog/Microsoft.Build.Framework.INodeLogger